GitCommands/CommitMessageManager.cs

```diff
@@ -90,7 +90,7 @@ public CommitMessageManager(Control owner, string workingDirGitDir, Encoding com
 
         internal CommitMessageManager(Control owner, string workingDirGitDir, Encoding commitEncoding, IFileSystem fileSystem, string? overriddenCommitMessage = null)
         {
-            ArgumentNullException.ThrowIfNull(nameof(owner));
+            ArgumentNullException.ThrowIfNull(owner);
```

Comment:
What is report if the owner is null?
Should the standard check be used instead?
Not requesting changes
```suggestion
            if (owner is null)
            {
                throw new ArgumentNullException(nameof(owner));
            }
```

Reply:
> What is report if the owner is null?

"System.ArgumentNullException : Value cannot be null. (Parameter 'owner')"

> Should the standard check be used instead?

Igor started to switch from
`owner ?? throw new ArgumentNullException`
to `ThrowIfNull`, which I endorse because it is the least redundant and a one-liner.

Reply:
In .NET 6 there are only few helper methods of this nature and many more were added in .NET 7 and 8.